packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.js

```diff
@@ -55,33 +60,24 @@ function TranscribedLines ({ lines, scale, task, theme }) {
           const mark = { length, x1, y1, x2, y2 }
           const id = `complete-${index}`
           return (
-            <ConsensusLine
-              role='img'
-              aria-describedby={id}
-              aria-label={line.consensusText}
-              key={line.id}
-              onClick={() => showConsensus(line)}
-              onKeyDown={e => (e.key === 'Enter' && showConsensus(line))}
-              tabIndex={0}
-            >
-              <TranscriptionLine
-                state='complete'
-                mark={mark}
-                scale={scale}
-              />
-              <Tooltip
-                background={theme.global.colors['light-6']}
-                className='tooltip'
-                id={id}
-                index={index}
-                label={{
-                  fill: theme.global.colors['neutral-6'],
-                  text: counterpart('TranscribedLines.complete')
-                }}
-                x1={x1}
-                y1={y1}
-              />
-            </ConsensusLine>
+            <Tooltip id={id} label={<TooltipLabel fill={fills.complete} label={counterpart('TranscribedLines.complete')} />}>
+              <ConsensusLine
+                role='img'
+                aria-describedby={id}
+                aria-label={line.consensusText}
+                focusColor={focusColor}
+                key={line.id}
+                onClick={() => showConsensus(line)}
+                onKeyDown={e => (e.key === 'Enter' && showConsensus(line))}
```

Comment:
I think out of scope, but noting we might want to add space `' '` to also `showConsensus(line)`

Reply:
This will be refactored with the implementation of the actual movable modal as well as updating the role.